dockers/docker-sonic-mgmt/Dockerfile.j2

```diff
@@ -169,3 +168,4 @@ RUN ~/lib/azure-cli/bin/python -m pip install azure-keyvault==0.3.7 -U
 # Install Virtual Environment
 RUN python -m virtualenv --system-site-packages env-201811
 RUN env-201811/bin/pip install ansible==2.0.0.2
+RUN pip install pycryptodome
```

Comment:
can you pin down the version?

Reply:
Sure. Pin'ed to the current version 3.9.8.

Reply:
Do you want to add in a note so that in future we remember that this was moved to the end for a reason?